redfish-core/lib/chassis.hpp

```diff
@@ -136,8 +136,8 @@ inline void getPhysicalSecurityData(std::shared_ptr<bmcweb::AsyncResp> aResp)
             {
                 // do not add err msg in redfish response, because this is not
                 //     mandatory property
-                BMCWEB_LOG_ERROR << "DBUS error: no matched iface " << ec
-                                 << "\n";
+                BMCWEB_LOG_INFO << "DBUS error: no matched iface " << ec
```

Comment:
Since we don't have /xyz/openbmc_project/Intrusion it is hitting here

Reply:
I don't think I see this one upstream.. Can we upstream this one?

Reply:
Hmm, I have this upstream at https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/48795/2/redfish-core/lib/chassis.hpp. Is that what you're asking or more about the intrusion sensor function?

Reply:
Got it. 
Nah, I am good. I missed it.